src/components/address-delivery/address-delivery.js

```diff
@@ -36,37 +36,56 @@ export default class AddressDelivery extends Component {
 
 	async handleSelectStreet(e) {
 		const streetsList = await this.pizzaServiceMock.getStreets();
-		const street = e.target.value;
 
+		this.handleInputChange(e);
 		this.setState({
 			streetsList,
-			street,
+		})
+	};
+
+	async handleSelectHouse(e) {
+		const housesList = await this.pizzaServiceMock.getHouses();
```

Comment:
Your `getStreets()` and `getHouses ` methods of pizzaServiceMock must mimic to interface of real  pizzaService, there are should be method's parameters for requests 
```suggestion
         	const streetsList = await this.pizzaServiceMock.getStreets(streetSearchQuery);
		const housesList = await this.pizzaServiceMock.getHouses(houseId);
```